src/main/java/org/aksw/simba/lemming/metrics/single/SingleValueMetricResult.java

```diff
@@ -0,0 +1,68 @@
+package org.aksw.simba.lemming.metrics.single;
+
+/**
+ * The class stores the metric name and the result which can be reused.
+ * 
+ * @author Atul
+ *
+ */
+public class SingleValueMetricResult implements UpdatableMetricResult {
```

Comment:
Where is the difference between this class and the `SimpleMetricResult` class? If both classes are the same, please keep only one of them and remove the other.

Reply:
Yes, they are similar. I have removed SingleValueMetricResult class.